moira.py

```diff
@@ -23,6 +23,15 @@
 _et_cache = {}
 
 
+def _to_bytes(s):
```

Comment:
I am not completely sure how necessary this function is, but doing `.encode()` instead of `_to_bytes` everywhere I used `_to_bytes` results in messages like `AttributeError: 'bytes' object has no attribute 'encode'. Did you mean: 'decode'?`.